ui/cypress.sh

```diff
@@ -1,16 +1,34 @@
 #!/usr/bin/env bash
 
+[[ "${IN_CI,,}" =~ ^(true)|(yes)|1$ ]] && IN_CI=true || IN_CI=''
+
 # On single node CONTROL_PLANE_IP = eth0
-CONTROL_PLANE_IP=$(ip a show eth0 | grep -Po "inet \K[\d.]+")
+CONTROL_PLANE_IP=$(/usr/sbin/ip a show eth0 | grep -Po "inet \K[\d.]+")
 
-# The version of Cypress we install below requires GTK3 to be available
-test -n "${IN_CI}" && sudo yum install -y gtk3
+if [[ $IN_CI ]]; then
+    # The version of Cypress we install below requires GTK3 to be available
+    packages=(
+        gtk3
+        xorg-x11-server-Xvfb
+        libXtst*
+        libXScrnSaver*
+        GConf2*
+        alsa-lib*
+        gcc-c++
+        make
+        nodejs
+    )
+    curl -sL https://rpm.nodesource.com/setup_10.x | sudo -E bash -
+    sudo yum install -y epel-release
+    sudo yum install -y "${packages[@]}"
+fi
 
-npm install --no-save --quiet --no-package-lock cypress@3.5.0 cypress-cucumber-preprocessor@1.12.0 cypress-wait-until@1.6.0
+npm install --no-save --quiet --no-package-lock cypress@3.5.0 \
+    cypress-cucumber-preprocessor@1.12.0 cypress-wait-until@1.6.0
```

Comment:
IIRC, this whole setup takes some time. Would be great to have a VM image we can easily spawn with dependencies pre-installed, just share it the test files and run them alongside a running cluster.